app/src/main/java/org/oppia/android/app/testing/NavigationDrawerTestActivity.kt

```diff
@@ -32,6 +32,11 @@ class NavigationDrawerTestActivity : InjectableAppCompatActivity(), RouteToTopic
     title = getString(R.string.menu_home)
   }
 
+  override fun onRestart() {
+    super.onRestart()
+    homeActivityPresenter.handleOnReStart()
```

Comment:
Rename this function to `handleOnRestart `.
`Restart` is correct and `ReStart` is incorrect.

Reply:
Done.